blockchain/src/history/history_store.rs

```diff
@@ -1083,7 +946,11 @@ mod tests {
     #[test]
     fn prove_num_leaves_works() {
         // Initialize History Store.
-        let env = VolatileDatabase::new(20).unwrap();
+        let env = MdbxDatabase::new_volatile(DatabaseConfig {
+            max_tables: Some(20),
```

Comment:
I think we should move that into the `Default::default()`, it's basically used everywhere…